Gems/WhiteBox/Code/Source/WhiteBoxToolApiReflection.cpp

```diff
@@ -93,12 +93,24 @@ namespace WhiteBox
                     {
                         return Api::MeshFaceCount(*WhiteBoxMeshFromHandle(*whiteBoxMeshHandle));
                     })
+                ->Method(
+                    "MeshVertexCount",
+                    [](WhiteBoxMeshHandle* whiteBoxMeshHandle)
+                    {
+                        return Api::MeshVertexCount(*WhiteBoxMeshFromHandle(*whiteBoxMeshHandle));
+                    })
```

Comment:
I like the added method but am curious - is this ever different than `MeshFaceCount*3`?

Reply:
I think there's a possibility, because there's implicit support between quads and tris in the APIs. This means we cannot be sure beforehand if the constraint `MeshFaceCount*3` will be met. Also, it's most likely less than that number, because faces can share vertices

Reply:
I think right now it actually probably is `MeshFaceCount * 3` but it _could_ changed in future, so it's likely useful to have both in case